lib/matplotlib/tests/test_axes.py

```diff
@@ -4669,7 +4669,7 @@ def test_rcparam_grid_minor():
         (('both'), (True, True)),
         (('major'), (True, False)),
         (('minor'), (False, True))
-        )
+    )
```

Comment:
please undo this change

Reply:
Done in https://github.com/matplotlib/matplotlib/pull/8690/commits/af603a93495230161eb3c2354236f986b15769f8